tests/utils/test_distributed.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+import torch.multiprocessing as mp
+from tests.test_utils import (
+    assert_expected,
+    gpu_test,
+    init_distributed_on_file,
+    set_rng_seed,
+    with_temp_files,
+)
+from torch import Tensor
+from torchmultimodal.utils.distributed import gather_tensor
+
+BATCH_SIZE = 4
+EMBEDDING_DIM = 8
+
+
+class TestGatherTensor:
+    """
+    Test gather_tensor method with backprop_in_gather param
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        set_rng_seed(1234)
+
+    @pytest.fixture()
```

Comment:
The parentheses are not required when there're no arguments: https://docs.pytest.org/en/6.2.x/fixture.html#back-to-fixtures

Reply:
Fixed in the latest revision. Thank you!